pages/docs/reference/fun-interfaces.md

```diff
@@ -0,0 +1,101 @@
+---
+type: doc
+layout: reference
+title: "Functional interfaces (SAM interfaces)"
+---
+
+# Functional (SAM) interfaces
+
+An interface with only one abstract method is called a _functional interface_, or a _Single Abstract 
+Method (SAM) interface_.
+
+Functional interfaces in Kotlin are marked with the `fun` keyword.
```

Comment:
Here it'd be more correct to use the word `modifier`:
```
... are marked with the `fun` modifier
```

Reply:
thanks. updated.